sites/website/src/docs/resources/cheat-sheet.md

```diff
@@ -0,0 +1,705 @@
+
+* Assembles the building blocks of `@microsoft/fast-foundation` to create its component set. 
+
+* Use this library when you want to integrate [FAST Components](#using-components) into an existing site or app.
+
+To install the components, use either `npm` or `yarn`:
+```shell
+npm install --save @microsoft/fast-components
+``` 
+```shell
+yarn add @microsoft/fast-components
+```
+
+To use a Web Component as a custom element in HTML, the custom element must be registered:
+
+```ts
+import { 
+    fastButton, 
+    provideFASTDesignSystem 
+} from "@microsoft/fast-components";
+
+provideFASTDesignSystem()
+    .register(
+        fastButton() // custom element
+    );
+```
+
+---
+### [@fluentui/web-components](https://www.fast.design/docs/components/getting-started)
+
+**A component library that implements Microsoft's [Fluent Design System](https://www.microsoft.com/design/fluent/#/).**
+
+* Assembles the building blocks of `@microsoft/fast-foundation` to create its component set. 
+
+* Use this library when you want components to look and feel like those found in Microsoft products.
+
+To install the components, use either `npm` or `yarn`:
+```shell
+npm install --save @fluentui/web-components
+``` 
+```shell
+yarn add @fluentui/web-components
+```
+
+To use a Web Component as a custom element in HTML, the custom element must be registered:
+
+```ts
+import { 
+    fluentButton, 
+    provideFluentDesignSystem 
+} from "@fluentui/web-components";
+
+provideFluentDesignSystem()
+    .register(
+        fluentButton() // custom element
+    );
+```
+
+---
+If you are attempting to configure the components for integration into a specific Microsoft product, the following table provides `AccentBaseColor` values you can use:
+
+| Product     | AccentBaseColor | 
+| :---        | :---            | 
+| Office      | #D83B01         | 
+| Word        | #185ABD         | 
+| Excel       | #107C41         | 
+| PowerPoint  | #C43E1C         | 
+| Teams       | #6264A7         | 
+| OneNote     | #7719AA         | 
+| SharePoint  | #03787C         | 
+| Stream      | #BC1948         | 
+
+---
+
+
+## [Using Components](https://www.fast.design/docs/components/getting-started)
+### Setup
+
+To register design system components:
+
+```ts
+import { 
+    fastButton, 
+    provideFASTDesignSystem 
+} from "@microsoft/fast-components";
+
+provideFASTDesignSystem()
+    .register(
+        fastButton() // desired custom component
+    );
+```
+
+Or, register all system components:
+
+```ts
+import { 
+    allComponents, 
+    provideFASTDesignSystem 
+} from "@microsoft/fast-components";
+
+provideFASTDesignSystem().register(allComponents);
+```
+
+With the components registered, add any component to the HTML:
+
+```html
+<fast-button>Hello world</fast-button>
+```
+
+---
+
+### [FAST Components](https://explore.fast.design/)
+
+| Component | Name | Component Explorer | 
```

Comment:
This feels hard to maintain. I would just defer to an authoritative source of all components like Explorer or Storybook rather than list them all again (for now) here.

Reply:
I agree, we list them several times in a few locations, they should only be located in one.

Reply:
@bheston @janechu Great points, thank you.  I will remove the table.

Would this make more sense?

Launch our [Component Explorer](https://explore.fast.design/) for a comprehensive list of FAST Components.